src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBERS_LIMIT = 100;
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(NUMBERS_LIMIT));
```

Comment:
We can add fields with type `Random` and `ColorSupplier` to our class, so we won't need to create instances every time